gridfs/grid_file.py

```diff
@@ -611,6 +612,33 @@ def readline(self, size=-1):
         data.seek(0)
         return data.read(size)
 
+    def readlines(self, hint=-1):
```

Comment:
IOBase implements  readlines() so we get it for free by subclassing. Well after we fix the `GridOut.__iter__` chunk vs line problem described above we will get the correct behavior for free.

Reply:
Removed the method :+1: